pilot/lib/pilot/build_manager.rb

```diff
@@ -116,11 +116,18 @@ def distribute_build(uploaded_build, options)
       if options[:distribute_external]
         external_group = TestFlight::Group.default_external_group(uploaded_build.provider_id, uploaded_build.app_id)
```

Comment:
You mentioned that not every account has this group. Should we check if `external_group` is nil here, and show an error message?